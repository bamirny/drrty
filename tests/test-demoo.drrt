def int main(){
    int i;
    int j;
    list[str] images;

    createHTMLDocument("printbig.css");
    makeHeader("\"Umm we've been lazy so we haven't been using Github...\"");
    createHTML("<h2 class=\"lazy-or-stupid\">\"That's not lazy, that's stupid!\" -Prof Edwards </h2>");

    images = ["1.jpeg","2.jpeg","3.jpeg","4.jpeg"];

    for (i = 0; i < 28; i = i + 1){
      if( j == 0){
        createElement("div", "", "bigClass"); 
        j = 1; 
      }else{
        if (i >= 0 && i <7){
            createElement("img", images.get(0), "smallClass");  
        }
        if (i >= 7 && i < 14){
            createElement("img", images.get(1), "smallClass");  
        }
        if (i >= 14 && i < 21){
            createElement("img", images.get(2), "smallClass");  
        }
        if (i >= 21 && i < 28){
            createElement("img", images.get(3), "smallClass");  
        }
        j = 0;
      }
  }
  return 0;


}