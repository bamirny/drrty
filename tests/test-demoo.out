<!DOCTYPE html><html><head><link rel="stylesheet" href="printbig.css"></head><body><h1>"Umm we've been lazy so we haven't been using Github..."</h1>
<h2 class="lazy-or-stupid">"That's not lazy, that's stupid!" -Prof Edwards </h2><img class="smallClass" src="1.jpeg">
<div class="bigClass"></div>
<img class="smallClass" src="1.jpeg">
<div class="bigClass"></div>
<img class="smallClass" src="1.jpeg">
<div class="bigClass"></div>
<img class="smallClass" src="1.jpeg">
<div class="bigClass"></div>
<img class="smallClass" src="2.jpeg">
<div class="bigClass"></div>
<img class="smallClass" src="2.jpeg">
<div class="bigClass"></div>
<img class="smallClass" src="2.jpeg">
<div class="bigClass"></div>
<img class="smallClass" src="3.jpeg">
<div class="bigClass"></div>
<img class="smallClass" src="3.jpeg">
<div class="bigClass"></div>
<img class="smallClass" src="3.jpeg">
<div class="bigClass"></div>
<img class="smallClass" src="3.jpeg">
<div class="bigClass"></div>
<img class="smallClass" src="4.jpeg">
<div class="bigClass"></div>
<img class="smallClass" src="4.jpeg">
<div class="bigClass"></div>
<img class="smallClass" src="4.jpeg">
<div class="bigClass"></div>
